docs/sections/guides.md

```diff
@@ -713,3 +647,459 @@ To re-run the entire workflow, use:
+
+In addition, you should be familiar with the concepts introduced in the 
+[Getting Started](https://popper.readthedocs.io/en/latest/sections/getting_started.html)
+section.
+This guide uses examples from machine learning but no prior knowledge of the field
+is required.
+
+By default, this guide assumes that you use the Docker container engine, but 
+highlights where the workflow will differ if you use another engine.
+
+### Getting started
+
+The examples presented in this guide come from a workflow developed for the 
+[Flu Shot Learning](https://www.drivendata.org/competitions/66/flu-shot-learning/) 
+research competition on Driven Data.
+This workflow shows examples of using Popper to automate common tasks in computational
+research with R:
+- downloading data
+- using R Markdown
+- fitting/simulating a model using `tidymodels`
+- visualizing the results with `ggplot2`
+- building a LaTeX paper with up-to-date results
+
+To help follow allong, see this 
+[repository](https://github.com/getpopper/popper-examples/tree/master/workflows/comp-research/rstudio) 
+with the final version of the workflow.
+To adapt the advice in this guide to your own project, get started
+with this [Cookiecutter template for Popper](https://github.com/getpopper/cookiecutter-popper-r).
+
+Initial project structure
+```
+├── LICENSE                                 
+├── README.md                <- The top-level README.
+├── data                     <- The original, immutable data dump.
+├── output             
+|   ├── models               <- Serialized models, predictions, model summaries.
+|   └── figures              <- Graphics created during analysis.
+├── paper                    <- Generated analysis as PDF, LaTeX.
+│   ├── paper.tex
+|   └── referenced.bib
+└── src                      <- R source code for this project.
+    ├── notebooks            <- RMarkdown notebooks.
+    ├── get_data.sh          <- Script for downloading the original data dump.
+    ├── models.py            <- Script defining models.
+    ├── predict.py           <- Script for generating model predictions.
+    └── evaluate_model.py    <- Script for generating model evaluation plots.
+```
+
+### Getting data
+
+Your workflow should automate downloading or generating data to ensure that it uses the correct,
+up-to-date version of the data. In this example, you can download data with a 
+simple shell script:
+
+```sh
+#!/bin/sh
+cd $1
+
+wget "https://s3.amazonaws.com/drivendata-prod/data/66/public/test_set_features.csv" --no-check-certificate
+wget "https://s3.amazonaws.com/drivendata-prod/data/66/public/training_set_labels.csv" --no-check-certificate
+wget "https://s3.amazonaws.com/drivendata-prod/data/66/public/training_set_features.csv" --no-check-certificate
+
+echo "Files downloaded: $(ls)"
+```
+
+Now, wrap this step using a Popper workflow. In a new file `wf.yml` at the root 
+of the folder,
+
+```yaml
+steps:
+- id: "dataset"
+  uses: "docker://jacobcarlborg/docker-alpine-wget"
```

Comment:
can we pin to a specific version? that would guard against `latest` being updated on dockerhub